test.lua

```diff
@@ -866,4 +885,53 @@ describe("Flow", function()
             ]])
         end)
     end)
+
+    describe("Flow.embed", function ()
+        local embedded_form = flow.make_gui(function ()
+            return gui.VBox{
+                gui.Label{label = "This is the embedded form!"},
+                gui.Field{name = "test2"},
+            }
+        end)
+        pending"raises an error if called outside of a form context"
+        test("returns a flow widget", function ()
+            test_render(function (p, _)
+                return gui.HBox{
+                    gui.Label{label = "asdft"},
+                    embedded_form:embed{ player = p, prefix = "theprefix" },
```

Comment:
```suggestion
                    embedded_form:embed{ player = p, name = "theprefix" },
```

Reply:
Thanks for the fix!

I think prefix is clearer, myself (ie: I prefer the way the unit test does it over the way the code defines it)

At the end of the day, it's your project. With that in mind, which would you prefer?

Reply:
I think I prefer `name`, for consistency with `ScrollableVBox`, since the prefixing is only an implementation detail, and because it might get confused with the actual prefix (`"\2" .. name .. "\2"`). But I'll accept this PR with either if you really prefer `prefix`.